src/_variables.less

```diff
@@ -76,6 +76,16 @@
 @keyBoxWhite: #bdb6b6;
 @diffblack: #686363;
 @diffwhite: #fdf4f4;
+@gradientPink: #e60550;
```

Comment:
See the "Begin variables IN styles/_key.less" and "END VARIABLES IN styles/_key.less"? Move your variables from in between those comments and add before your variables similar comments before and after them but change _key.less to _gradient.less